packages/extension-koni-ui/src/components/Modal/ChangeVersionWalletContractModal.tsx

```diff
@@ -0,0 +1,196 @@
+// Copyright 2019-2022 @subwallet/extension-koni-ui authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { ResponseGetAllTonWalletContractVersion } from '@subwallet/extension-base/types';
+import { GeneralEmptyList } from '@subwallet/extension-koni-ui/components';
+import ChangeVersionWalletConractItem from '@subwallet/extension-koni-ui/components/ChangeVersionWalletConractItem';
+import { CHANGE_VERSION_WALLET_CONTRACT } from '@subwallet/extension-koni-ui/constants/modal';
+import { useGetAccountByAddress, useNotification } from '@subwallet/extension-koni-ui/hooks';
+import useTranslation from '@subwallet/extension-koni-ui/hooks/common/useTranslation';
+import { tonAccountChangeWalletContractVersion, tonGetAllWalletContractVersion } from '@subwallet/extension-koni-ui/messaging';
+import { ThemeProps } from '@subwallet/extension-koni-ui/types';
+import { TonWalletContractVersion } from '@subwallet/keyring/types';
+import { Button, Icon, SwList, SwModal } from '@subwallet/react-ui';
+import CN from 'classnames';
+import { CaretLeft, CheckCircle, FadersHorizontal } from 'phosphor-react';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import styled from 'styled-components';
+
+type Props = ThemeProps & {
+  onCancel?: VoidFunction;
+  id: string;
+  chainSlug: string;
+  address: string;
+};
+
+export type WalletContractItem = {
```

Comment:
rename to TonWalletContractItemType, then move to TonWalletContractItem component